server/models/LendingRequest.js

```diff
@@ -0,0 +1,7 @@
+import { Model } from 'objection';
+
+class lendingRequest extends Model {
+    static tableName = 'lending_requests';
+}
+
+export default lendingRequest;
```

Comment:
All models in the app start with a capital letter. Connvension

Reply:
I would fix this in the next push to resolve this branch. Thank you

Reply:
Fix this before I merge.

Reply:
Regarding the failing tests, I communicated with Kelvin and it would be fixed after his new PR is merged into the master branch, since the error is from the test on a feature he worked on.